src/holofuel/pages/Profile/Profile.js

```diff
@@ -43,7 +45,7 @@ export default function Profile () {
         <CopyAgentId agent={{ id }} isMe>
           <HashAvatar seed={id} styleName='avatar-image' data-testid='host-avatar' />
         </CopyAgentId>
-        <h3 styleName='nickname-display' data-testid='profile-nickname'>{nickname || 'Your Nickname'}</h3>
+        <h3 styleName='nickname-display' data-testid='profile-nickname'>{nickname || optimisitcNickname || 'Your Nickname'}</h3>
```

Comment:
`optimisticNickname` should override the `nickname` that comes from the query, so it should appear first in the `||` chain.

Reply:
This is true. Updating now.